src/App.js

```diff
@@ -2,6 +2,11 @@ import React, { useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import styles from "./App.module.css";
+<<<<<<< HEAD
+import Sidebar from "./Sidebar";
+=======
+>>>>>>> 42c54d3d54e4a7961743e7440745f8cd7f66b313
```

Comment:
Make sure to remove the following lines from the file:

```
<<<<<<< HEAD
=======
>>>>>>> 42c54d3d54e4a7961743e7440745f8cd7f66b313
```

Reply:
Yes, I will do it. Thank you :)